woocommerce-services.php

```diff
@@ -914,32 +876,33 @@ public function attach_hooks() {
 			$this->taxjar->init();
 			$this->paypal_ec->init();
 
-			// Primary condition to initiate shipping.
+			/*
+			 * Only register shipping label-related logic if WC Shipping is not active.
+			 */
 			if ( ! $this->is_wc_shipping_activated() ) {
-				add_action( 'rest_api_init', array( $this, 'wc_api_dev_init' ), 9999 );
```

Comment:
Removed instead of loading. See: https://github.com/Automattic/woocommerce-services/pull/2812/files#r1753874334